api/r/cellxgene.census/LICENSE.md

```diff
@@ -1,6 +1,6 @@
 # MIT License
 
-Copyright (c) 2023 Chan Zuckerberg Initiative Foundation
+Copyright (c) 2023-2024 Chan Zuckerberg Initiative Foundation
```

Comment:
Not sure if we want to uniform this to 2022-2024. I guess the R APIs were released in 2023, but this could be confusing.